crates/revm/src/interpreter/contract.rs

```diff
@@ -180,13 +109,16 @@ impl Contract {
 
 /// Mapping of valid jump destination from code.
 #[derive(Clone, Debug, Eq, PartialEq)]
+#[cfg_attr(feature = "with-serde", derive(serde::Serialize, serde::Deserialize))]
 pub struct ValidJumpAddress {
-    first_gas_block: u64,
-    analysis: Vec<AnalysisData>,
+    pub first_gas_block: u32,
+    /// Rc is used here so that we dont need to copy vector. We can move it to more suitable more accessable structure
+    /// without copying underlying vec.
+    pub analysis: Rc<Vec<AnalysisData>>,
```

Comment:
Can we make this an `Arc`? Forge has some shared backend logic that sends `AccountInfo` across thread bounds and the `Rc` makes it `!Send` :smile:

Reply:
^+1 we need this